aws/sdk/integration-tests/using-native-tls-instead-of-rustls/Cargo.toml

```
@@ -0,0 +1,24 @@
+[package]
+name = "using-native-tls-instead-of-rustls"
+version = "0.1.0"
+authors = ["AWS Rust SDK Team <aws-sdk-rust@amazon.com>"]
+edition = "2021"
+
+# See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
+
+[dev-dependencies]
+# aws-config pulls in rustls and several other things by default. We have to disable defaults in order to use native-tls
+# and then manually bring the other defaults back
+aws-config = { path = "../../build/aws-sdk/sdk/aws-config", default-features = false, features = [
+  "native-tls",
+  "rt-tokio",
+] }
+# aws-sdk-s3 brings in rustls by default so we disable that in order to use native-tls only
+aws-sdk-s3 = { path = "../../build/aws-sdk/sdk/s3", default-features = false, features = [
+  "native-tls",
+] }
+# aws-sdk-sts is the same as aws-sdk-s3
+aws-sdk-sts = { path = "../../build/aws-sdk/sdk/sts", default-features = false, features = [
+  "native-tls",
+] }
+tokio = { version = "1.20.1", features = ["full"] }
```

I think we can change this to just bring in the test utils, although it may also require the `rt` feature

```suggestion
tokio = { version = "1.20.1", features = ["test-util"] }
```

---

Yes. Addressed in 86896e5. The minimal feature set required seems to be `rt` and `macros`.